pkgs/applications/misc/cool-retro-term/default.nix

```
@@ -38,7 +46,7 @@ stdenv.mkDerivation rec {
     '';
     homepage = https://github.com/Swordifish90/cool-retro-term;
     license = with stdenv.lib.licenses; [ gpl2 gpl3 ];
```

```suggestion
    license = stdenv.lib.licenses.gpl3Plus;
```

Source: https://github.com/Swordfish90/cool-retro-term/blob/master/app/qml/AboutDialog.qml#L97